File: StockMarketAppBackend/StockMarketAppBackend/Stock.cs
namespace StockMarketAppBackend
{
    public class Stock
    {
        public string Name { get; set; }
        public float PriceNow { get; set; }

        public List<float> Diff { get; set; }

        public Stock(string name, float priceNow, List<float> diff)
        {
            Name = name;
            PriceNow = priceNow;
            Diff = diff;
        }

        public Stock()
        {

        }
    }
}

File: StockApp/app/model/Stock.js
Ext.define('stockMarketApp.model.Stock', {
    extend: 'stockMarketApp.model.Base',

    store: 'stockMarketApp.store.StockStore',
    fields: [
        {name: 'name', type: 'string'},
        {name: 'priceNow', type: 'float'},
        {name: 'diff', type: 'float'}
    ]
});
